packages/web-components/fast-foundation/src/select/select.ts

```diff
@@ -372,6 +378,12 @@ export class Select extends FormAssociatedSelect {
         super.slottedOptionsChanged(prev, next);
         this.setProxyOptions();
         this.updateValue();
+
+        if (this.collapsible) {
+            this.options.forEach(o => {
+                Observable.getNotifier(o).subscribe(this, "content");
```

Comment:
Whenever there is a manual subscribe call there usually needs to be a corresponding unsubscribe call.